framework/src/org/apache/cordova/CordovaInterfaceImpl.java

```diff
@@ -147,6 +151,8 @@ public void onSaveInstanceState(Bundle outState) {
             String serviceName = activityResultCallback.getServiceName();
             outState.putString("callbackService", serviceName);
         }
+
+        outState.putBundle("plugin", pluginManager.onSaveInstanceState());
```

Comment:
Do you need to check if pluginManager is not null? I ask because there is a check for that on line 116.